tabpy-tools/tools_tests/test_rest.py

```diff
@@ -52,7 +52,7 @@ def test_GET_InvalidData(self):
         data = {'cat'}
         try:
             self.assertEqual(self.rnw.GET(url, data), 'json')
-        except:
+        except Exception:  # TODO: refactor this...
```

Comment:
This is a rather strange test design to catch an Exception and make assertions about the type of exception raised. Would be better to use `self.assertRaises`

Reply:
Agreed, we are slowly reworking tests as well